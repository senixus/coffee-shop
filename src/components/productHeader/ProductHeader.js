import React from "react";

import Navbar from "../navbar/Navbar";

import "./productHeader.scss";

const ProductHeader = ({ title }) => {
  return (
    <header className="product-header">
      <Navbar />
      <div className="product-header__child">
        <h1>{title}</h1>
      </div>
    </header>
  );
};

export default ProductHeader;
