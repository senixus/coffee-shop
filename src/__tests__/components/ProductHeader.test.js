import { shallow } from "enzyme";
import React from "react";
import ProductHeader from "../../components/productHeader/ProductHeader";

import { create } from "react-test-renderer";

//const productHeader = create(<ProductHeader />);

it("expect to render ProductHeader component", () => {
  const mockTitle = "Product Header";

  expect(shallow(<ProductHeader title={mockTitle} />)).toMatchSnapshot();
  // expect(productHeader.toJSON()).toMatchSnapshot();
});
