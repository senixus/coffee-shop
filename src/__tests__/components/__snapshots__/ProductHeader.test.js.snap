// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ProductHeader component 1`] = `
<header
  className="product-header"
>
  <Navbar />
  <div
    className="product-header__child"
  >
    <h1>
      Product Header
    </h1>
  </div>
</header>
`;
